.github/workflows/CI-raiwidgets-pytest.yml

```diff
@@ -7,7 +7,7 @@ on:
     branches: [main]
 
 jobs:
-  ci-raiwidgets-python-typescript:
+  ci-raiwidgets-pytest:
     env:
```

Comment:
If this is just running pytest, then could you move raiwidgets under CI-python.yml?

Reply:
I can't actually, since it's required to do `yarn install` and `yarn buildall` first to build `raiwidgets`. That's why it was a separate pipeline to begin with.

Reply:
they used to all be in one pipeline but raiwidgets was too flaky and it also took much more time than the others, hence it was moved into its own pipeline

Reply:
see PR for context:
https://github.com/microsoft/responsible-ai-toolbox/pull/1158
the description on the PR is really good

Reply:
Thanks for linking! Exactly. That's why they should remain separate from CI-python.

Reply:
Why do we need to do `yarn install` and `yarn buildall` to run pytest for raiwidgets? These are python SDK tests so should be able to  run without `yarn`

Reply:
Great question! I was wondering that myself a few weeks ago until I realized that we want to test this particular version of raiwidgets on the PR branch. So if you look into the raiwidgets folder there's no JS code there. To get it there we need to run `yarn buildall` and then install the python package. If you don't do the yarn part then it'll fail as soon as you call the dashboard. If you're interested I can walk you through the handover between the two at some point 🙂

Reply:
Makes sense. I had a hunch but wasn't entirely sure. Thanks for clarifying this.